bluetooth: fast_pair: fmdn: Fix Kconfig dependency misalignment (#21946)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth/services/fast_pair.rst

    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_RING_REQ_TIMEOUT_DULT_BT_GATT` - The option configures the ringing timeout for connected peers that use DULT-based ringing mechanism.
      This option can only be used when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT` is enabled.
    * :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_RING_REQ_TIMEOUT_DULT_MOTION_DETECTOR` - The option configures the ringing timeout for ringing requests from the DULT motion detector.

>       This option can only be used when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` is enabled.

Reviewer: ```suggestion
      You can use this option only when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` Kconfig option is enabled.
```

Reviewer: This is not yet fixed.

Reviewer: You could leave this as is. I will fix this later here and other places, if needed.

Author: Thanks! I also see that the "Kconfig option" after option name is missing in multiple places in this file.

>       This option can only be used when the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_DULT_MOTION_DETECTOR` is enabled.

Reviewer 2: Please add a changelog entry.

Is the changelog for changes targeting the main branch only ready? @divipillai?

Reviewer: yeah, you can update the changelog here : https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst. It is cleaned for the next release.
